packages/mobile/src/send/Send.tsx

```diff
@@ -118,20 +117,12 @@ function Send({ route }: Props) {
         }
       )
 
-      if (multiTokenUseSendFlow) {
-        navigate(Screens.SendAmount, {
-          recipient,
-          isOutgoingPaymentRequest,
-          origin: isOutgoingPaymentRequest ? SendOrigin.AppRequestFlow : SendOrigin.AppSendFlow,
-        })
-      } else {
-        navigate(Screens.SendAmountLegacy, {
```

Comment:
i think we can delete the `SendAmountLegacy` component now? seems like nobody is navigating to it anymore

Reply:
Yes! 🙌